vuepress/docs/.vuepress/next.js

```diff
@@ -20,6 +20,37 @@ module.exports = {
                         title: 'Entando Architecture',
                         path: path + 'getting-started/concepts-overview.md',
                     },
+                    {
+                        title: 'Entando CLI',
+                        children: [
+                            {
+                                title: 'Basics',
+                            path: path + 'getting-started/entando-cli.md',
+                            },
+                            {
+                                title: 'ent Bundle Management',
```

Comment:
Do you think lowercase ent looks okay in the left nav when everything else is capitalized? I thought we discussed dropping ent it in the nav and just having Bundle Management, API Management, etc. since the context should be obvious.